docs/sdk/sdk-intro.md

```diff
@@ -0,0 +1,42 @@
+# Threeport SDK Introduction
+
+The Threeport SDK is a command line tool to enable platform engineers to rapidly
+develop extensions to Threeport.
+
+## How It Works
+
+A platform engineer uses the Threeport SDK to build software that provides
+custom abstractions and functionality for their development teams to use.
+
+Once the requirements are well understood, the process to build a custom
+Threeport extension is as follows:
+
+1. Initialize a Threeport extension project.  The platform engineer starts a new
+   project with its own git repo and uses the `threeport-sdk` CLI tool to
+   initialize the repository which scaffolds all the directories and code needed
+   to compile their software.
+1. Create one or more new API objects.  The engineer designs the data model
+   based on the requirements for the extensions.  The platform engineer uses the
+   `threeport-sdk` to create new API objects.  The boilerplate for the source
+   code is created by the SDK.  Then the engineer adds the fields that define
+   the objects attributes to meet the project's requirements.
+1. The boilerplate and scaffolding provided includes functions to add the
+   functionality for the reconcilers that will manage the state declared by
+   users in the API objects made available.  This reconciliation logic will be
```

Comment:
```suggestion
1. Create one or more new API objects.  The platform engineer designs a data
   model based on extension requirements and uses `threeport-sdk` to generate
   its boilerplate code.  Then the engineer updates this code with fields that
   define the object's attributes to meet the project's requirements.
1. The generated boilerplate code includes scaffolding to add functionality for
   reconcilers that will manage state of the new custom object.  This
   reconciliation logic will be compiled into one or more custom controllers
   that will be added to the Threeport control plane.
```

Reply:
I misused "scaffolding" and "boilerplate" here.  I've updated separately to fix.